packages/components/src/frame-provider/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { CacheProvider } from '@emotion/core';
+import createCache from '@emotion/cache';
+import memoize from 'memize';
+
+const memoizedCreateCacheWithContainer = memoize( ( container ) => {
+	return createCache( { container } );
+} );
+
+export default function FrameProvider( { children, iframeDocument } ) {
```

Comment:
Could this be renamed to emotion cache provider, perhaps?

Reply:
Maybe more `StyleFrameProvider` or something, I don't think we should be tied to a specific library.

Reply:
It provides the cache right? `StyleCacheProvider`?

Reply:
it is what makes the styles injection work, I  prefer `StyleProvider`

Reply:
`StyleProvider` might imply that it is necessary for basic style injection, but it's not. I think including the fact that this is specifically for `iframe`s is wise, like `StyleFrameProvider` or `IFrameStyleProvider` if we want to be the most descriptive.

Reply:
What do you think @ellatrix

Reply:
Sure, I just wanted to point out that it's not providing an iframe but rather the style cache. Maybe `StylesForIframeProvider`?

Not a blocker, I don't care that much :)